docs/reference/migration/migrate_6_0/plugins.asciidoc

```diff
@@ -47,6 +47,8 @@ See {plugins}/repository-azure-usage.html#repository-azure-repository-settings[A
 * The region setting has been removed. This includes the settings `cloud.aws.region`
 and `cloud.aws.ec2.region`. Instead, specify the full endpoint.
 
+* All `cloud.aws.*` and `cloud.aws.ec2.*` settings have been removed. Use `discovery.ec2.*` settings instead.
```

Comment:
Just a thought (nothing important TBH), may be replace with:

All `cloud.aws.*` and `cloud.aws.ec2.*` settings have been renamed to `discovery.ec2.*`.